dev/com.ibm.ws.security.mp.jwt.1.2_fat/fat/src/com/ibm/ws/security/mp/jwt12/fat/sharedTests/MPJwt12MPConfigTests.java

```diff
@@ -0,0 +1,647 @@
+     *            - null when running a good test, the expectations to check in
+     *            the case of a bad/negative test
+     * @throws Exception
+     */
+
+    public void standardTestFlow(LibertyServer server, String rootContext,
+                                 String theApp, String className) throws Exception {
+        standardTestFlow(MpJwt12FatConstants.SIGALG_RS256, server, rootContext, theApp, className, MpJwt12FatConstants.AUTHORIZATION, MpJwt12FatConstants.TOKEN_TYPE_BEARER, null);
+
+    }
+
+    public void standardTestFlow(LibertyServer server, String rootContext,
+                                 String theApp, String className, Expectations expectations) throws Exception {
+        standardTestFlow(MpJwt12FatConstants.SIGALG_RS256, server, rootContext, theApp, className, MpJwt12FatConstants.AUTHORIZATION, MpJwt12FatConstants.TOKEN_TYPE_BEARER,
+                         expectations);
+
+    }
+
+    public void standardTestFlow(LibertyServer server, String rootContext, String theApp, String className, String location, String name,
+                                 Expectations expectations) throws Exception {
+        standardTestFlow(MpJwt12FatConstants.SIGALG_RS256, server, rootContext, theApp, className, location, name, expectations);
+
+    }
+
+    public void standardTestFlow(String builder, LibertyServer server, String rootContext,
+                                 String theApp, String className) throws Exception {
+        standardTestFlow(builder, server, rootContext, theApp, className, MpJwt12FatConstants.AUTHORIZATION, MpJwt12FatConstants.TOKEN_TYPE_BEARER, null);
+
+    }
+
+    public void standardTestFlow(LibertyServer server, String rootContext, String theApp, String className, String location, String name) throws Exception {
+        standardTestFlow(MpJwt12FatConstants.SIGALG_RS256, server, rootContext, theApp, className, location, name, null);
+    }
+
+    public void standardTestFlow(String builder, LibertyServer server, String rootContext, String theApp, String className, String location, String name) throws Exception {
+        standardTestFlow(builder, server, rootContext, theApp, className, location, name, null);
+    }
+
+    public void standardTestFlow(String builder, LibertyServer server, String rootContext, String theApp, String className, String location, String name,
+                                 Expectations expectations) throws Exception {
+        String testUrl = buildAppUrl(server, rootContext, theApp);
+        standardTestFlow(builder, testUrl, className, location, name, expectations);
+    }
+
+    public void standardTestFlow(String builder, String testUrl, String className, String location, String name) throws Exception {
+        standardTestFlow(builder, testUrl, className, location, name, null);
+    }
+
+    public void standardTestFlow(String builder, String testUrl, String className, String location, String name, Expectations expectations) throws Exception {
```

Comment:
Having a separate class kinda like we did with `TestSettings` would be useful for setting all of this information and avoiding nine overloaded `standardTestFlow` methods.

Reply:
1.1 and 1.2 have their own standardTestFlow methods (they really do have to do different things).
Have reduced the overloaded methods.